install/merger/merger_test.go

```diff
@@ -0,0 +1,55 @@
+package merger
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type DataImpl struct {
+	labels      map[string]string
+	updateCount int
+	mergeCount  int
+}
+
+func NewData() *DataImpl {
+	return &DataImpl{
+		labels: map[string]string{
+			"label": "value",
+		},
+	}
+}
+
+func (d *DataImpl) Labels() *map[string]string {
```

Comment:
In such a case I would rather use mock than fake. Mocks have nice `AssertCalled()` and `AssertNotCalled()` methods.